deploy.sh

```diff
@@ -1,24 +1,27 @@
 #!/bin/sh
 
 # Scripts to deploy application to staging server
-# @todo #45:30min Continue implementing deploy scripts. Find out user 
-#  and domain of the staging server and replace scp and ssh with correct data.
-#  Add scripts in cron (like the one created in #47). Verify the web 
-#  application is running (execute couple fo curl requests).
+# @todo #126:30min Finish implementing deploy scripts. Add scripts in cron 
+#  (like the one created in #47). Verify the web application is running - 
+#  execute couple of curl requests.
 
 echo "-- Run tests"
 pytest
 
 echo "-- Creating staging tag"
 git tag -a staging-$tag -m "Rultor deploy staging-$tag"
 
+STAGING_SERVER=$(jq -r '.credentials.server.staging.address' ../credentials.json)
+STAGING_USER=$(jq -r '.credentials.server.staging.username' ../credentials.json)
+STAGING_KEY=../staging.id_rsa
+
 echo "-- Copy application code to staging server"
-scp -r . user@staging-server:/app
+scp -i $STAGING_KEY -r . $STAGING_USER@$STAGING_SERVER:/app
 
 # add scripts in cron (like the one created in #47)
 # verify the webapplication is running
-ssh user@staging-server << EOF
-  echo "-- Creating database user: timless_user"
+ssh -i $STAGING_KEY $STAGING_USER@$STAGING_SERVER << EOF
+  echo "-- Creating database user: timeless_user"
```

Comment:
should we take db user from credentials.json as well?

Reply:
@vryazanov Yes, of course, make sense.